### DISCUSSION TITLE
Carousel, hover affects all other carousel

### DISCUSSION THREAD

**Discussion by sqveeze** [mantinedev/mantine#2021] [Q&A]
Hello,

I really happy for that mantine has carousel package now, after an upgrade i just removed my past carousel package and tryin to use the mantine one. Everything works fine except the hovering.

According to this example in the docs: https://mantine.dev/others/carousel/#show-controls-on-hover

I have added the required styles, however i have a list of cards and inside each card i have a carousel rendered. Now when i hover over one carousel in a card, all other carousel start showing the controls. Is there any way to show controls on only on the hovered card?

**Comment by rtivital** [comment]
It does not happen on codesandbox – https://codesandbox.io/s/frosty-blackwell-tcfy0z?file=/src/App.tsx
